app/builders/subscriber_auth_email_builder.rb

```diff
@@ -25,23 +25,11 @@ def call
   attr_reader :subscriber, :destination, :token
 
   def subject
-    "Confirm your email address"
+    manage_your_subscriptions_subject
   end
 
   def body
-    <<~BODY
-      # Click the link to confirm your email address
-
-      ^ [Confirm your email address](#{link})
-
-      You need to do this to manage your GOV.UK email subscriptions. The link will stop working in 7 days.
-
-      # Didn’t request this email?
-
-      Ignore or delete this email if you didn’t request it. Your subscriptions won’t be changed.
-
-      [Contact GOV.UK](https://www.gov.uk/contact/govuk) if you have any problems with your email subscriptions.
-    BODY
+    manage_your_subscriptions_body
```

Comment:
Can we put the body text directly into this method? Do we need the extra method call?